homeassistant/components/synology_dsm/services.yaml

```diff
@@ -0,0 +1,15 @@
+# synology-dsm service entries description.
+
+reboot:
+  description: Reboots the NAS.
+  fields:
+    serial:
+      description: serial of NAS to reboot; optional when only one is configured.
```

Comment:
```suggestion
      description: serial of the NAS to reboot; required when multiple NAS are configured.
```

I see it the other way around. What do you think of that ?

Reply:
your suggestion sounds much more convenient